openrtb2/bid_request.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // 3.2.1 Object: BidRequest
```

Comment:
Pull request [Add AutoRefresh and AutoRefreshSettings to Bid Request](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/pull/40) of the [openrtb2.x/2.6.md](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/develop/2.6.md) repository lists `AutoRefresh` and `AutoRefreshSettings` under the `BidRequest` list but, this fields, along with `DurFlooors` are not in the table depicted in the [3.2.1 - Object: BidRequest](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/7e8693acdd8db71c760bd43c5460e2c52ec8be1c/2.6.md#321---object-bidrequest-) section. Are they getting added to the table, and eventually, as fields of the `BidRequest` struct eventually? Should we add them to the `BidRequest` object already?

Reply:
I'm curious what pointed your attention to that specific commit? That was a commit to a develop branch. The location was moved to the Imp object before release.

> Are they getting added to the table, and eventually, as fields of the BidRequest struct eventually? 

No.

Reply:
The [main branch](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md) correctly places the refresh object in the desired location.